tracking/archive.hpp

```diff
@@ -44,7 +44,10 @@ struct Restrictions
 
 struct Limits
 {
-  Limits() = delete;
+  Limits(double minLat, double maxLat, double minLon, double maxLon)
+    : m_minLat(minLat), m_maxLat(maxLat), m_minLon(minLon), m_maxLon(maxLon)
+  {
+  }
```

Comment:
Do we need additional ctor here?

Reply:
Yes. Prior to C++20, the compiler would generate a deduction guide for a struct if it had no user-defined constructors, allowing the struct to be initialized using an initializer list. However, in C++20, a user-defined constructor is required to use aggregate initialization with an initializer list. This change could cause the code to fail to compile if the struct being initialized does not have a user-defined constructor.

We can leave the struct as
```cpp
struct Limits
{
 double m_minLat;
 double m_maxLat;
 double m_minLon;
 double m_maxLon;
};
```

And it won't require any additional ctor. But originally default ctor was deleted, so I decided to create an explicit ctor.

Reply:
There are two possible solutions:
```cpp
// 1
struct Limits
{
 double m_minLat;
 double m_maxLat;
 double m_minLon;
 double m_maxLon;
};
// 2
struct Limits
{
 Limits(double minLat, double maxLat, double minLon, double maxLon);
 double m_minLat;
 double m_maxLat;
 double m_minLon;
 double m_maxLon;
};
```

I implemented the second one because originally default ctor was deleted.
But we can switch to the first solution

Reply:
We try to check the old code and its usage if there is a need to touch it. The final change should ideally take into account the usage, not just blindly copy whatever properties the code had before. Because there were so many changes, and these properties may not be needed anymore.